### ISSUE TITLE
entry.touch() doesn't work

### ISSUE THREAD

**Issue by BoomerB** [libkeepass/pykeepass#120]
```
$ python
Python 2.7.15rc1 (default, Nov 12 2018, 14:31:15) 
[GCC 7.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from pykeepass import PyKeePass
>>> kp = PyKeePass('test.kdbx', 'monilove')
>>> group_name = 'Robot'
>>> entry_name = 'KPserver'
>>> group = kp.find_groups(name=group_name, first=True)
>>> item = kp.find_entries(group=group, title=entry_name, first=True)
>>> item.touch(modify=True)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/pykeepass/entry.py", line 219, in touch
    self._element.Times.LastAccessTime = datetime.utcnow()
AttributeError: 'lxml.etree._Element' object has no attribute 'Times'
```

**Comment by BoomerB** [comment]
I submitted a pull request some time ago
#114

**Comment by Evidlo** [comment]
Please add a test.

**Comment by Evidlo** [comment]
Did you ever make a PR for this?  It seems like they're all closed.

**Comment by BoomerB** [comment]
Hi,
I tried several times but it never really worked. last time I tried to
contribute, I wanted to run the checks on my changes before I create a pull
request but all tests failed for some other changes. I waited for a while
in the hopes that the one who caused the problems would fix them and then I
was busy with other things.
I think I still have the code somewhere. I can try to organize it into a
few small pull requests. Then it might be easier to get them through

Kind regards
Bernhard

Am Do., 12. Sept. 2019 um 04:05 Uhr schrieb Evan Widloski <
notifications@github.com>:
